tests/Behat/Context/FeatureContext.php

```diff
@@ -16,39 +16,77 @@ class FeatureContext extends SilverStripeContext
      */
     public function iShouldSeeAListOfVersions()
     {
-        $page = $this->getSession()->getPage();
-        $versionsTable = $page->find('css', '.history-viewer .table');
-        assertNotNull($versionsTable, 'I should see a list of versions');
+        $this->getVersions();
     }
 
     /**
      * @Then I should see a list of versions in descending order
      */
     public function iShouldSeeAListOfVersionsInDescendingOrder()
     {
-        $page = $this->getSession()->getPage();
-        $versions = $page->findAll('css', '.history-viewer .table tbody tr td:nth-of-type(1)');
+        $versions = $this->getVersions();
+        $this->clickVersion(current($versions));
```

Comment:
I'm not sure I get the click event here, if you're just looking for an ordered list?

Reply:
You're right - this has nothing to do with "seeing a list of versions in descending order"